tests/ending_blank_line.rs

```diff
@@ -43,3 +43,19 @@ fn incorrect_files() {
         testdir.test_command_fail_with_args(args, expected_output);
     }
 }
+
+#[test]
+fn two_blank_lines_at_the_end() {
+    let content = "ABC=DEF\nD=BAR\nFOO=BAR\n\n";
```

Comment:
Maybe will we move this test to a separate file?
And will write some tests for other cases?

Reply:
OK (this test is related to both issues, so I put it here).

Reply:
I did it and found the bug with a comment between blank lines )

Now I don't skip comments, so I have added more tests.

Reply:
Please, give an example.

Reply:
This example triggered the false `ExtraBlankLine` warning (I already fixed it):

```

# comment

ABC=DEF

```

This test checks it:
https://github.com/dotenv-linter/dotenv-linter/blob/528e98864dadb9c636f7d28b3076786d23f70ebb/tests/extra_blank_line.rs#L11